ddtrace/internal/ci_visibility/writer.py

```diff
@@ -167,6 +171,12 @@ def recreate(self):
             timeout=self._timeout,
             dogstatsd=self.dogstatsd,
             sync_mode=self._sync_mode,
+            report_metrics=self._report_metrics,
```

Comment:
CIVis.recreate() was missing a bunch of arguments, so we were losing state when this this method was used. This issue did not show up in CI since we were using `tracer.configure(...)`. Now that many civis tests have been updated to use `Tracer._recreate()` this method needs to do the right thing 🫡

Reply:
Thank you Munir!